packages/url/src/test/url.ts

```diff
@@ -35,14 +35,16 @@ describe('isValidUrl', () => {
 
 describe('withProtocol', () => {
   it('should add protocol correctly if applicable', () => {
-    expect(withProtocol('https://foo-bar.com/test.jpg')).toBe(
-      'https://foo-bar.com/test.jpg'
-    );
-    expect(withProtocol('http://foo-bar.com/test.jpg')).toBe(
-      'http://foo-bar.com/test.jpg'
-    );
-    expect(withProtocol('foo-bar.com/test.jpg')).toBe(
-      'https://foo-bar.com/test.jpg'
-    );
+    expect(withProtocol('https://foo.com/test')).toBe('https://foo.com/test');
+    expect(withProtocol('http://foo.com/test')).toBe('http://foo.com/test');
+    expect(withProtocol('foo.com/test')).toBe('https://foo.com/test');
```

Comment:
The only reason for this change is to make each fit on a single line because it's prettier 😂